modules/manage/pages/kubernetes/k-decommission-brokers.adoc

```diff
@@ -77,17 +79,17 @@ ID    HOST                          PORT   RACK
 ```
 ====
 
-The output shows four racks (A/B/C/D), so you might want to have at least four brokers to make use of all racks.
+The output shows four racks (A/B/C/D), so you might want to have at least four brokers to use all racks.
 
 Rack awareness is just one aspect of availability. Refer to xref:deploy:deployment-option/self-hosted/kubernetes/k-high-availability.adoc[High Availability] for more details on deploying Redpanda for high availability.
 
 === Cost
 
-Infrastructure costs increase with each broker, so adding a broker means an additional instance cost. For example, if you deploy Redpanda on GKE on https://gcloud-compute.com/n2-standard-8.html[n2-standard-8^] GCP instances, the instance cost of the cluster is $1925 per month. Reducing the number of brokers to five would save $550 per month, and reducing it further to three brokers would save $1100 per month. You must also consider other costs, but they won't be as impacted by changing the broker count.
+Infrastructure costs increase with each broker, so adding a broker means an additional instance cost. For example, if you deploy Redpanda on GKE on https://gcloud-compute.com/n2-standard-8.html[n2-standard-8^] GCP instances, the instance cost of the cluster is $1925 per month. Reducing the number of brokers to five would save $550 per month, and reducing it to three brokers would save $1100 per month. You must also consider other costs, but they won't be as impacted by changing the broker count.
```

Comment:
What is the mechanism to insure these listed costs stay up-to-date?

Reply:
I've adjusted this paragraph to call out that these are hypothetical examples.